contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -524,9 +560,35 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
         return (key_, signature, _index < operators[_operator_id].usedSigningKeys);
     }
 
+
+    function getSigningKeys(uint256 _operator_id, uint256 _offset, uint256 _limit)
+        external
+        view
+        operatorExists(_operator_id)
+        returns (
+            bytes memory pubkeys,
+            bytes memory signatures,
+            bool [] memory used
+        )
+    {
+        require(_offset + _limit <= operators[_operator_id].totalSigningKeys, "OUT_OF_RANGE");
```

Comment:
Better using safemath here (replace `+` with `add`) to avoid overflows

Reply:
fixed